src/documents/forms/EditDocumentForm.js

```diff
@@ -66,12 +86,12 @@ const EditDocumentForm = React.forwardRef(
           }) => (
             <>
               {showFieldHints &&
```

Comment:
these conditionals are getting a little long (my fault) can we see if there's a better way to maybe pluck the values from objects and do the checks that way?